src/Auth/Guards/SessionGuard.php

```diff
@@ -22,6 +22,14 @@ class SessionGuard extends GuardContract
      */
     private array $provider;
 
+    /**
+     * Defines the session_key
+     *
+     * @var array
+     */
+    private string $session_key;
```

Comment:
Where are you going to use the `session_key`?  It's defined but not used

Reply:
Using dynamic properties is now deprecated, retourn error php 8.2